tests/functional/firstrun_sign_up.js

```diff
@@ -59,6 +59,33 @@ define([
         })
 
         .then(FunctionalHelpers.testIsBrowserNotified(self, 'fxaccounts:can_link_account'))
+
+        // user should be transitioned to the choose what to Sync page
+        .findByCssSelector('#fxa-choose-what-to-sync-header')
```

Comment:
Is Fx Desktop able to accept web CWTS data already? If so, great, if not, we'll have to disable CWTS for the firstrun flow, and when the Fx patch lands, enable CWTS for firstrun, maybe using a different context?